src/assets/wise5/components/aiChat/ai-chat-student/ai-chat-student.component.ts

```diff
@@ -20,6 +23,8 @@ import { AiChatComponent } from '../AiChatComponent';
 })
 export class AiChatStudentComponent extends ComponentStudent {
   component: AiChatComponent;
+  computerAvatar: ComputerAvatar;
```

Comment:
Add access modifier?
```suggestion
  protected computerAvatar: ComputerAvatar;
```